src/box/memtx_space.c

```diff
@@ -276,7 +276,7 @@ memtx_space_replace_all_keys(struct space *space, struct tuple *old_tuple,
 	 * Don't use MVCC engine for ephemeral in any case.
```

Comment:
Nit: I refer to this subsystem as `memtx`, not `txm` (just to be on the same page).